### PR TITLE
chore: run linting in compass package

### PR DESCRIPTION
**Pull request by addaleax**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [x] Misc

## Types of changes

- [ ] *Backport Needed*
- [x] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass/package.json [hunk 1]
```diff
@@ -141,7 +141,7 @@
     "evergreen-expansions": "hadron-build info --format=yaml --flatten > expansions.yml",
     "build-info": "hadron-build info",
     "jsx": "babel ./src --out-dir ./lib",
-    "lint": "eslint",
+    "lint": "eslint .",
```
Comment (addaleax) [inline packages/compass/package.json:144]: This was missing previously … :slightly_smiling_face:

File: packages/compass/src/main/logging.ts [hunk 2]
```diff
@@ -194,7 +194,7 @@ async function showLogFileDialog(logFilePath: string) {
     case 3: {
       try {
         const tempFilePath = await extractPartialLogFile(logFilePath);
-        shell.openPath(tempFilePath);
+        await shell.openPath(tempFilePath);
```
Comment (addaleax) [inline packages/compass/src/main/logging.ts:197]: … and this is the only code change of substance :slightly_smiling_face:

**Review by Anemy — APPROVED** [review]
nice catch

**Comment by addaleax** [comment]
> nice catch - makes me think there could be other places we're missing the `.` thinking

I think we usually do something like

```js
    "eslint": "eslint",
    "prettier": "prettier",
    "lint": "npm run eslint . && npm run prettier -- --check .",
```

where omitting the `.` is fine, I guess the compass package was just special this way? With a quick grep I couldn’t find other problems of this type, at least :slightly_smiling_face:

**Pull request merged**